### PR TITLE
Created new Pull Request Template

### PR DESCRIPTION
**Pull request by ashish4321**
#19 Fixed

### CODE REVIEW

**Comment by ashish4321** [comment]
@shiv07tiwari @DreamyPhobic Pls check it

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Issue Number :
```
Comment (DreamyPhobic) [inline .github/PULL_REQUEST_TEMPLATE.md]: change issue number to Closes #

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ -
+ 
+# Purpose
```
Comment (DreamyPhobic) [inline .github/PULL_REQUEST_TEMPLATE.md]: Change it to "What changes are introduced to solve this?"

**Comment by ashish4321** [comment]
Pls review

**Pull request merged**